baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -241,3 +345,127 @@ def build_message_handler(self) -> KombuMessageHandler:
+
+    For simple cases where you just need a basic queue with all the default
+    parameters for your message broker, you can use `KombuQueueConsumerFactory.new`.
+
+    If you need more control, you can create the :py:class:`~kombu.Queue` s yourself and
+    use the constructor directly.
+    """
+
+    def __init__(
+        self,
+        baseplate: Baseplate,
+        name: str,
+        connection: kombu.Connection,
+        queues: Sequence[kombu.Queue],
+        handler_fn: BatchHandler,
+        error_handler_fn: Optional[BatchErrorHandler] = None,
+        health_check_fn: Optional[HealthcheckCallback] = None,
+        serializer: Optional[KombuSerializer] = None,
+        batch_size: int = 1,
+        batch_timeout: timedelta = timedelta(0, 1, 0, 0, 0, 0, 0),
+    ) -> None:
+        """`KombuQueueConsumerFactory` constructor.
+
+        :param baseplate: The Baseplate set up for your consumer.
+        :param exchange: The `kombu.Exchange` that you will bind your :py:class:`~kombu.Queue` s
+            to.
+        :param queues: List of  :py:class:`~kombu.Queue` s to consume from.
+        :param queue_name: Name for your queue.
+        :param routing_keys: List of routing keys that you will create :py:class:`~kombu.Queue` s
+            to consume from.
+        :param handler_fn: A function that will process an individual message from a queue.
+        :param error_handler_fn: A function that will be called when an error is thrown
+            while executing the `handler_fn`. This function will be responsible for calling
+            `message.ack` or `message.requeue` as it will not be automatically called by
+            `KombuMessageHandler`'s `handle` function.
+        :param health_check_fn: A `baseplate.server.queue_consumer.HealthcheckCallback`
+            function that can be used to customize your health check.
+        :param serializer: A `baseplate.clients.kombu.KombuSerializer` that should
+            be used to decode the messages you are consuming.
+        """
+        self.baseplate = baseplate
+        self.connection = connection
+        self.queues = queues
+        self.name = name
+        self.handler_fn = handler_fn
+        self.error_handler_fn = error_handler_fn
+        self.health_check_fn = health_check_fn
+        self.serializer = serializer
+        self.batch_size = batch_size
+        self.batch_timeout = batch_timeout
+
+    @classmethod
+    def new(
+        cls,
+        baseplate: Baseplate,
+        exchange: kombu.Exchange,
+        connection: kombu.Connection,
+        queue_name: str,
+        routing_keys: Sequence[str],
+        handler_fn: BatchHandler,
+        error_handler_fn: Optional[BatchErrorHandler] = None,
+        health_check_fn: Optional[HealthcheckCallback] = None,
+        serializer: Optional[KombuSerializer] = None,
+        batch_size: int = 1,
+        batch_timeout: timedelta = timedelta(0, 1, 0, 0, 0, 0, 0),
```

Comment:
Same here as above, you can just write this as:

```suggestion
        batch_timeout: timedelta = timedelta(seconds=1),
```